src/generic/Matrix.jl

```diff
@@ -28,32 +28,23 @@ export MatrixSpace, fflu!, fflu, solve_triu, isrref, charpoly_danilevsky!,
 #
 ###############################################################################
 
-function similar(x::Mat{T}, R::Ring=base_ring(x)) where T <: RingElement
-   TT = elem_type(R)
-   M = similar(x.entries, TT)
-   for i in 1:size(M, 1)
-      for j in 1:size(M, 2)
-         M[i, j] = zero(R)
-      end
-   end
-   z = MatSpaceElem{TT}(M)
-   z.base_ring = R
-   return z
-end
-
-function similar(x::Mat{T}, R::Ring, r::Int, c::Int) where T <: RingElement
+function _similar(x::Union{Mat{T},MatAlgElem{T}}, R::Ring, r::Int, c::Int) where T <: RingElement
```

Comment:
There is a space missing after the comma (,)

Reply:
OK, I didn't realize it was the code style in this repo.

Reply:
Ouch, I thought I had pushed the fix 1 hour ago!